Cargo.toml

```diff
@@ -2159,13 +2159,13 @@ wasm = false
 name = "fallible_systems"
 path = "examples/ecs/fallible_systems.rs"
 doc-scrape-examples = true
+required-features = ["bevy_mesh_picking_backend"]
```

Comment:
Is this needed? Example seems to work on main before this fix and I see `bevy_mesh_picking_backend` in `default`.

Reply:
strictly speaking, no, it's a default feature